tests/integration/integrations/feast/feature_stores/test_feast_feature_store.py

```diff
@@ -1,46 +0,0 @@
-#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
```

Comment:
What is the motivation for removing Feast tests?

Reply:
@strickvl

Reply:
These were already commented out + we weren't running them.